### DISCUSSION TITLE
Databag

### DISCUSSION THREAD

**Discussion by n-hebert** [selfhst/icons#46] [New Icon Request]
Official repo and site: https://github.com/balzack/databag

**Comment by shollyethan** [comment]
Databag has been added. I wasn't able to find a high-quality image, so I'm open to updating it if anyone has a better asset.

**Reply by n-hebert** [reply]
Thanks!

The high res shortage is possibly because it's a WIP, as I just found out here: https://github.com/balzack/databag/blob/main/.design%2FDESIGN.md
